src/components/DashboardAnalytics/ProfitSharingAnalytics/ProfitSharingAnalytics.js

```diff
@@ -0,0 +1,173 @@
+import ProfitSharingEquityChart from "./ProfitSharingEquityChart";
+import "./ProfitSharingAnalytics.scss";
+
+/**
+ * @typedef {import("../../../services/tradeApiClient.types").ProfitSharingBalanceHistory} ProfitSharingBalanceHistory
+ * @typedef {import("../../../services/tradeApiClient.types").ProviderEntity} ProviderEntity
+ */
+
+/**
+ * @typedef {Object} DefaultProps
+ * @property {ProviderEntity} provider
+ */
+
+/**
+ * Render analytics panels for profit sharing providers.
+ *
+ * @param {DefaultProps} props Component props.
+ * @returns {JSX.Element} JSX
+ */
+const ProfitSharingAnalytics = ({ provider }) => {
+  const storeSession = useStoreSessionSelector();
+  const [performance, setPerformance] = useState(null);
+  const [performanceLoading, setPerformanceLoading] = useState(false);
+  const [balanceHistory, setBalanceHistory] = useState(
+    /** @type {ProfitSharingBalanceHistory} */ (null),
+  );
+  const [balanceHistoryLoading, setBalanceHistoryLoading] = useState(false);
+  const storeSettings = useStoreSettingsSelector();
+
+  const dispatch = useDispatch();
+
+  const getProviderPerformance = () => {
+    const payload = {
+      token: storeSession.tradeApi.accessToken,
+      providerId: provider.id,
+    };
+    setPerformanceLoading(true);
+
+    tradeApi
+      .providerPerformanceGet(payload)
+      .then((response) => {
+        setPerformance(response);
+      })
+      .catch((e) => {
+        dispatch(showErrorAlert(e));
+      })
+      .finally(() => {
+        setPerformanceLoading(false);
+      });
+  };
+  useEffect(getProviderPerformance, []);
+
+  const getProfitSharingBalanceHistory = () => {
+    const payload = {
+      token: storeSession.tradeApi.accessToken,
+      providerId: provider.id,
+      exchangeInternalId: storeSettings.selectedExchange.internalId,
+    };
+    setBalanceHistoryLoading(true);
+
+    tradeApi
+      .getProfitSharingBalanceHistory(payload)
+      .then((response) => {
+        setBalanceHistory(response);
+      })
+      .catch((e) => {
+        dispatch(showErrorAlert(e));
+      })
+      .finally(() => {
+        setBalanceHistoryLoading(false);
+      });
+  };
+  useEffect(getProfitSharingBalanceHistory, []);
+
+  return (
+    <Box
+      alignItems="center"
+      className="profitSharingAnalytics"
+      display="flex"
+      flexDirection="column"
+      justifyContent="center"
+    >
+      {performanceLoading ? (
+        <CircularProgress color="primary" size={50} />
+      ) : (
+        performance && <TradingPerformance performance={performance} />
+      )}
+      {balanceHistoryLoading ? (
+        <Box
+          alignItems="center"
+          display="flex"
+          flexDirection="column"
+          justifyContent="center"
+          mt="50px"
+        >
+          <CircularProgress color="primary" size={50} />
+        </Box>
+      ) : (
+        balanceHistory && (
+          <>
+            <TotalEquityBar>
```

Comment:
@cwagner22  this is what I was trying to avoid, I mean everywhere we want to use this, we will have to write this "TotalEquityBar" whole block again, I know we are avoiding duolication of css. what do you think?

Reply:
I tried to make it as reusable as possible so now we only need to use the EquityPart components. If you compare to the original version in AvailableBalance.js, the code is actually shorter, and there is not much duplication, just reusing the EquityPart components. It also take care of the markup for the usdt conversion info and pnl.
And wrapped with TotalEquityBar but this one has now become just a few lines of code to handle mostly the styling.
Let me know what you think, or we can compare with your version.